release/models/ospf/openconfig-ospf-types.yang

```diff
@@ -98,6 +104,72 @@ module openconfig-ospf-types {
   }
 
   // identities
+  identity MESSAGE_DIGEST_ENCRYPTION {
```

Comment:
This is not "encryption" - these are hash algorithms for authentication

Reply:
renamed to use MESSAGE_DIGEST_ALGO